packages/react/src/components/CommandBar/CommandBar.types.ts

```diff
@@ -116,6 +116,20 @@ export interface ICommandBarProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   ariaLabel?: string;
 
+  /**
+   * When using farItems, primaryGroupAriaLabel and secondaryGroupAriaLabel function as
+   * labels for each group that are exposed to screen reader users.
+   * This helps clarify when a screen reader user is entering or leaving each group.
+   */
+  primaryGroupAriaLabel?: string;
+
+  /**
+   * When using farItems, primaryGroupAriaLabel and secondaryGroupAriaLabel function as
+   * labels for each group that are exposed to screen reader users.
+   * This helps clarify when a screen reader user is entering or leaving each group.
+   */
+  secondaryGroupAriaLabel?: string;
```

Comment:
I wonder if this should be something like `farItemsGroupAriaLabel` to match the `farItems` prop?

Reply:
ah, sure!